### PR TITLE
Update cadvisor to 6116f265302357cbb10f84737af30b1f13ce2d6c

### PR DESCRIPTION
**Pull request by Random-Liu**
Update cadvisor to 6116f265302357cbb10f84737af30b1f13ce2d6c.

For
* containerd integration performance optimization: https://github.com/google/cadvisor/commit/1dd94694690a56df4d8b7bf38db3e1d1bd887976
* overlay bug fix: https://github.com/google/cadvisor/commit/812cc819acc5ebc353b9992f43411f5b750d313c

@kubernetes/sig-node-pr-reviews @dashpole 
/cc @thockin for vendor review.

Signed-off-by: Lantao Liu <lantaol@google.com>

**Release note**:

```release-note
none
```

### CODE REVIEW

File: Godeps/OWNERS [hunk 1]
```diff
@@ -1,4 +0,0 @@
-reviewers:
```
Comment (dashpole) [inline Godeps/OWNERS]: any idea why it is making this change?

**Review by dashpole — COMMENTED** [review]
cadvisor bump
/lgtm

File: Godeps/OWNERS (hunk 1 above)
Comment (Random-Liu) [reply Godeps/OWNERS]: My bad. Will add it back

Comment (Random-Liu) [reply Godeps/OWNERS]: Done.

**Pull request merged**